File: src/main/java/net/softbell/bsh/iot/dto/bshp/v1/ItemValueV1DTO.java
package net.softbell.bsh.iot.dto.bshp.v1;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : BSHPv1 전용 Item Value DTO
 */
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class ItemValueV1DTO
{
	private byte pinId;
	private short pinStatus;
}

File: src/main/kotlin/net/softbell/bsh/iot/dto/bshp/v1/ItemValueV1Dto.kt
package net.softbell.bsh.iot.dto.bshp.v1

/**
 * @author : Bell(bell@softbell.net)
 * @description : BSHPv1 전용 Item Value DTO
 */
data class ItemValueV1Dto (
    var itemIndex: Byte,
    var itemStatus: Double?
)